tests/foreman/ui/test_settings.py

```diff
@@ -209,53 +211,58 @@ def test_positive_selectors(session):
                 setting_cleanup(setting_attr, str(original_val.default))
 
 
-@stubbed()
 @tier3
-def test_positive_update_login_page_footer_text_with_long_string():
-    """Attempt to update parameter "Login_page_footer_text"
-        with long length string under General tab
+def test_positive_update_login_page_footer_text_with_long_string(session):
+    """Testing to update parameter "Login_page_footer_text with long length
+    string under General tab
 
     :id: b1a51594-43e6-49d8-918b-9bc306f3a1a2
 
-    :steps:
-
-        1. Navigate to Administer -> settings
-        2. Click on general tab
-        3. Input long length string into login page footer field
-        4. Assert value from login page
-
     :expectedresults: Parameter is updated
 
     :CaseImportance: Medium
 
-    :CaseAutomation: notautomated
-
     :CaseLevel: Acceptance
     """
+    property_name = 'login_text'
+    property_value = [
+        ele.value
+        for ele in entities.Setting().search(
+            query={'search': 'name={}'.format(property_name)}
+        )
+    ][0]
+    login_text_data = gen_string('alpha', 270)
```

Comment:
anything about 270 char size?

Reply:
The purpose of this test case is to check the "login page footer text" updated with long strings or not, and that's the only reason I used string length.